ai-platform-unified/notebooks/unofficial/reduction_server/distributed-training-reduction-server.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>w/o a "gs://" prefix the gsutil commands below will not work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>